app/backend/resources/schema.edn

```diff
@@ -1817,7 +1817,81 @@
           {:db/ident :land-owner-opinion/authority-position
            :db/valueType :db.type/string
            :db/cardinality :db.cardinality/one
-           :db/doc "Rich text containing the authoritys position on the opinion"}]]}]
+           :db/doc "Rich text containing the authoritys position on the opinion"}]]}
+
+ {:db/ident :migration/contracts
+  :txes [[{:db/ident :contract/number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to contract file in document management system for procurement part"}
+
+          {:db/ident :contract/procurement-number
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to State procurement registry"}
+
+          {:db/ident :contract/thk-procurement-link
+           :db/valueType :db.type/string
+           :db/cardinality :db.cardinality/one
+           :db/doc "Link to THK procurement, not editable"}
+
+          ;; Contract statuses
+          {:db/ident :contract.status/contract-signed
+           :enum/attribute :contract/status}
+          {:db/ident :contract.status/in-progress
+           :enum/attribute :contract/status}
+          {:db/ident :contract.status/warranty-period
+           :enum/attribute :contract/status}
+          {:db/ident :contract.status/completed
+           :enum/attribute :contract.response/status}
+
+          {:db/ident :contract/status
+           :db/valueType :db.type/ref
+           :db/cardinality :db.cardinality/one
+           :db/doc "Enum reference to contract status"}
+
+          {:db/ident :contract/region
```

Comment:
the region is a selection in the designs. https://www.figma.com/proto/uVnwU2YMZrHEHlKSLuJ4ol/Contracts?node-id=45%3A513&viewport=-341%2C-236%2C0.2963973581790924&scaling=min-zoom

Should we add that in some way in to the schema? Since projects probably should share the same region as well?

Reply:
Enum of all regions in Estonia? Can be independent entity without impacting this object, which may have region as free text.